text/slugify.ts

```diff
@@ -0,0 +1,441 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+
+const CHAR_MAP = new Map([
```

Comment:
This list looks random and arbitrarily chosen. How about starting with something very minimal? For example:

```
const CHAR_MAP = new Map([
  ["$", "dollar"],
  ["%", "percent"],
  ["&", "and"],
]);
```

Reply:
This is the list taken from `npm:slugify`. I even wonder if we should implement char replacement at all. There exists no standard and any list will always be arbitrary. Maybe replacements should be done by the user before calling `slugify()` and all non-valid chars then are removed?

Reply:
I agree with your suggestion, Tim.

Reply:
No char replacement makes more sense to me